docker/create_cluster.py

```diff
@@ -70,22 +65,48 @@ def create_cluster(clean, network_name, number_of_nodes, public_key_path):
     # Loop through nodes, start them on the network that was just created
     print('Starting {} containers on network {} ...'.format(number_of_nodes, network_name))
     container_names = ['{}.{}'.format(network_name, i) for i in range(number_of_nodes)]
```

Comment:
This will be difficult at the moment due to the fact that we are reusing the containers for different purposed dependent on the cluster configuration. We are not aware of the cluster configuration when running these.